pkg/apis/etcd/v1beta2/backup_types.go

```diff
@@ -84,7 +84,7 @@ type BackupSource struct {
 
 // BackupPolicy defines backup policy.
 type BackupPolicy struct {
-	// timeout is the maximal time of retriving plus saving an etcd backup.
+	// timeout is the maximal allowed time in second of the entire backup process.
 	Timeout int64 `json:"timeout,omitempty"`
```

Comment:
After a second thought, maybe we should rename this TimeoutInSecond or TimeoutSecond

Reply:
`TimeoutInSecond` seems good.